### ISSUE TITLE
"Error problem loading plugin @lando/vuepress-theme-default-plus" on all lando commands

### ISSUE THREAD

**Issue by hades200082** [lando/lando#3384]
Lando version: 3.6.4
OS: Tried on both Windows 10 and Windows 11

Both myself and several work colleagues get the following error when running pretty much any lando command in powershell, even just running `lando version`:

```
ERROR ==> problem loading plugin @lando/vuepress-theme-default-plus from C:/snapshot/cli/node_modules/@lando/vuepress-theme-default-plus/index.js: TypeError: Cannot read property 'options' of undefined
    at C:\snapshot\cli\node_modules\@lando\vuepress-theme-default-plus\index.js
    at Plugins.load (C:\snapshot\cli\lib\plugins.js)
    at C:\snapshot\cli\lib\lando.js
    at Array.map (<anonymous>)
    at bootstrapConfig (C:\snapshot\cli\lib\lando.js)
    at bootstrapRouter (C:\snapshot\cli\lib\lando.js)
    at C:\snapshot\cli\lib\lando.js
    at tryCatcher (C:\snapshot\cli\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:729:18)
    at Promise._fulfill (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:673:18)
    at Promise._resolveCallback (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:466:57)
    at Promise._settlePromiseFromHandler (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:559:17)
    at Promise._settlePromise (C:\snapshot\cli\node_modules\bluebird\js\release\promise.js:604:18)
```

The command _appears_ to complete properly for me after the error but the error is causing some confusion amongst my colleagues who are less familiar with lando and docker.

**Comment by aohlrogge** [comment]
I'm having this exact same issue.

**Comment by labboy0276** [comment]
Please [urge your containers and start fresh: https://docs.lando.dev/help/purging-containers.html it seems you have some containers that are "stuck"

**Comment by bryanbuchs** [comment]
I've got the same erro message, right down to the exact paths. I have no `C:/snapshot`, and this started today after I updated to the latest release.

**Comment by josequevedo1990** [comment]
> Please purge your containers and start fresh: https://docs.lando.dev/help/purging-containers.html it seems you have some containers that are "stuck"

I already did it, but I'm still getting the same error, do you have any other workaround?

**Comment by labboy0276** [comment]
@josequevedo1990 try this `rm -rf ~/.lando` if you have any special config in there, please save it first.  Then do a `lando rebuild` on your site.

If that doesn't work, try upgrading to the [latest pre-release 3.6.5](https://github.com/lando/lando/releases/tag/v3.6.5).  And purging your containers again.

If that doesn't work, then please tell me what OS you are on and what version you upgraded from.

**Comment by labboy0276** [comment]
Also I should add if you are on windows, the store app config may be located at `C:\Users\ME\.lando` you can use `lando config | grep userConfRoot` to locate the exact path and I recommend either removing the folder completely or just the cache folder in it.

**Comment by josequevedo1990** [comment]
@labboy0276 Thanks a lot, it has now worked successfully

**Comment by labboy0276** [comment]
awesome @josequevedo1990 what worked for you?

**Comment by josequevedo1990** [comment]
@labboy0276 This is what I did:
1. I ran this command first `lando config | grep userConfRoot`
2. I removed `C:\Users\ME\.lando` directory
3. Upgraded to the [latest pre-release 3.6.5](https://github.com/lando/lando/releases/tag/v3.6.5)
4. Purged my containers 

After that, I didn't see the error again

**Comment by hades200082** [comment]
3.6.5 fixes it for me.

**Comment by labboy0276** [comment]
OK, I will close this out as it seems there is a workaround / fix to the issue.

**Comment by oldeb** [comment]
I also confirm that upgrading 3.6.5 fixed the issue **without** having to purge the containers or removing the .lando directory.

**Comment by poppabear8883** [comment]
Upgrading is what caused the error for me. I have tried the suggestions above and the issue remains for me. I'm on windows and I removed the `.lando` directory and purged containers then `lando start`  in my project and the error remains.

**Comment by Mohamed207470** [comment]
3.6.5 fixes it for me. thank you

**Comment by theMusician** [comment]
Upgrading straight from 3.6.4 to 3.6.5 fixed this for me as well.

**Comment by naticap** [comment]
> @josequevedo1990 try this `rm -rf ~/.lando` if you have any special config in there, please save it first. Then do a `lando rebuild` on your site.
> 
> If that doesn't work, try upgrading to the [latest pre-release 3.6.5](https://github.com/lando/lando/releases/tag/v3.6.5). And purging your containers again.
> 
> If that doesn't work, then please tell me what OS you are on and what version you upgraded from.

I also got those error, and just install latest pre released versionThanks 3.6.5. And its worked. Thanks @labboy0276

**Comment by billyws** [comment]
Upgrading to the [latest pre-release 3.6.5](https://github.com/lando/lando/releases/tag/v3.6.5) worked for me. Also, I'm just curious if there was a fix for Lando version 3.6.4
